app/src/androidTest/java/com/example/sharingang/DetailedItemFragmentTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.sharingang
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
```

Comment:
For this class, I think it might be cleaner to import Espresso.method, instead of calling that method with Espresso.method in the tests